File: Modul 6/Soal 2/PRAK502-2210817220018-Piola Evania.py
def hitung(nilai1, nilai2):
    return nilai1 - nilai2

def mutlak(angka):
    return abs(angka)

for i in range(3):
    a, c, b, d = map(int, input().split())
    Hasil = hitung(a, b) + hitung(c, d)
    print(mutlak(Hasil))
    print("\n")

File: Modul 6/Soal 2/PRAK502-2210817220018-Piola-Evania.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
    return nilai1 - nilai2;
}

int mutlak(int angka){
    return abs(angka);
}

int main(){
    for(int i = 1; i < 4; i++){
        int a, b, c, d;
        scanf("%d", &a);
        scanf("%d", &c);
        scanf("%d", &b);
        scanf("%d", &d);
        int Hasil = hitung(a, b) + hitung(c, d);
        printf("%d\n\n", mutlak(Hasil));
    }
    return 0;
}
